src/promptflow-azure/promptflow/azure/_storage/cosmosdb/collection.py

```diff
@@ -39,17 +39,17 @@ class CollectionCosmosDB:
     def __init__(self, span: Span, is_cloud_trace: bool, created_by: Dict[str, Any]):
         self.span = span
         self.created_by = created_by
-        self.collection_name = span.session_id
+        self.collection_name = span.resource.get(SpanResourceAttributesFieldName.COLLECTION, TRACE_DEFAULT_COLLECTION)
```

Comment:
We need to get collection name from resource.attributes. Or local to cloud trace will be broke.
I will fix in this PR.